apps/src/code-studio/components/SoundCategory.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import * as color from "../../util/color";
```

Comment:
Here (and in other files) you should be able to just do `import color from '../../util/color'` or what i usually do, `import color from "@cdo/apps/util/color";`